Kernel/RndLiteralSelector.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of the source code of the software program
+ * Vampire. It is protected by applicable
+ * copyright laws.
+ *
+ * This source code is distributed under the licence found here
+ * https://vprover.github.io/license.html
+ * and in the source directory
+ */
+/**
+ * @file RndLiteralSelector.hpp
+ * Defines class RndLiteralSelector.
+ */
+
+
+#ifndef __RndLiteralSelector__
+#define __RndLiteralSelector__
+
+#include "Forwards.hpp"
+#include "Lib/SmartPtr.hpp"
+#include "Ordering.hpp"
+
+#include "LiteralSelector.hpp"
+
+namespace Kernel {
+
+/**
+ * Class SpassLiteralSelector implements literal
+ * selectors as understood from the code of SPASS 3.7.
+ */
```

Comment:
Oh, wrong comment (copy-paste error). Adding the one that should have been there.

And yes, we have `SpassLiteralSelector` just next to this class and it would make sense to add the proper explanation there, but I have honestly forgotten since 2016...